### PR TITLE
Add corsika DB parameter schema

### PR DESCRIPTION
**Pull request by GernotMaier**
This PR adds schema files for those CORSIKA parameters which have been added to the model parameter repository with [this merge request](https://gitlab.cta-observatory.org/cta-science/simulations/simulation-model/model_parameters/-/merge_requests/43).

See also #1056 for some additional information.

### CODE REVIEW

File: simtools/schemas/model_parameters/corsika_particle_kinetic_energy_cutoff.schema.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    allowed_range:
+      min: 0.05
+    required: true
+  - name: electrons
+    type: float64
+    description: Kinetic energy cutoffs for electrons.
+    unit: GeV
+    default: 0.020
+    allowed_range:
+      min: 0.05
```
Comment (tobiaskleiner) [inline simtools/schemas/model_parameters/corsika_particle_kinetic_energy_cutoff.schema.yml:34]: min should be lower than default, did you mean 0.005?

File: simtools/schemas/model_parameters/corsika_particle_kinetic_energy_cutoff.schema.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    required: true
+  - name: gammas
+    type: float64
+    description: Kinetic energy cutoffs for gammas.
+    unit: GeV
+    default: 0.020
+    allowed_range:
+      min: 0.05
```
Comment (tobiaskleiner) [inline simtools/schemas/model_parameters/corsika_particle_kinetic_energy_cutoff.schema.yml:42]: same here

**Review by tobiaskleiner — COMMENTED** [review]
Thanks @GernotMaier, see my comments.

File: simtools/schemas/model_parameters/corsika_particle_kinetic_energy_cutoff.schema.yml (hunk 1 above)
Comment (GernotMaier) [reply simtools/schemas/model_parameters/corsika_particle_kinetic_energy_cutoff.schema.yml]: Well spotted! Corrected.


Screenshot 2024-07-11 at 13 51 03

File: simtools/schemas/model_parameters/corsika_particle_kinetic_energy_cutoff.schema.yml (hunk 2 above)
Comment (GernotMaier) [reply simtools/schemas/model_parameters/corsika_particle_kinetic_energy_cutoff.schema.yml]: Fixed.

File: simtools/schemas/model_parameters/corsika_particle_kinetic_energy_cutoff.schema.yml (hunk 1 above)
Comment (tobiaskleiner) [reply simtools/schemas/model_parameters/corsika_particle_kinetic_energy_cutoff.schema.yml]: Ok good! Interesting it is that low for electrons and photons.

**Review by tobiaskleiner — APPROVED** [review]

File: simtools/schemas/model_parameters/corsika_particle_kinetic_energy_cutoff.schema.yml (hunk 1 above)
Comment (GernotMaier) [reply simtools/schemas/model_parameters/corsika_particle_kinetic_energy_cutoff.schema.yml]: Yes - but note that this is for CORSIKA and some detectors are sensitive to such low energies. For Cherenkov, the emission threshold is 20 MeV at sea level.

**Pull request merged**